packages/paste-core/components/button/src/DestructiveSecondaryButton.tsx

```diff
@@ -14,28 +14,26 @@ import {DirectButtonPropTypes} from './proptypes';
 const defaultStyles: BoxStyleProps = merge(BaseStyles.default, {
   color: 'colorTextLinkDestructive',
   backgroundColor: 'colorBackgroundBody',
-  boxShadow: 'shadowBorderDestructive',
+  boxShadow: 'shadowBorderWeak',
   _hover: {
-    color: 'colorTextLinkDestructiveStronger',
-    backgroundColor: 'colorBackgroundDestructiveWeakest',
-    boxShadow: 'shadowBorderDestructiveStronger',
+    color: 'colorTextLinkDestructive',
+    backgroundColor: 'colorBackgroundBody',
+    boxShadow: 'shadowBorderDestructive',
   },
   _focus: {
-    color: 'colorTextLinkDestructiveStronger',
-    backgroundColor: 'colorBackgroundDestructiveWeakest',
     boxShadow: 'shadowFocus',
```

Comment:
issue: missing inner border, this probably needs to be `shadowFocusShadowBorder`

<img width="223" alt="image" src="https://user-images.githubusercontent.com/368249/222012043-230c8dbf-780e-4cb7-9355-6531a9e101f2.png">